packages/main/src/TimePickerClock.ts

```diff
@@ -0,0 +1,991 @@
+ * Depending on settings, the clock can display only outer set of items (when the clock displays hours in 12-hour mode,
+ * minutes or seconds), or outer and inner sets of items (when the clock displays hours in 24-hours mode).
+ * The step for displaying or selecting of items can be configured.
+ *
+ * <code>ui5-time-picker-clock</code> is used as part of <code>ui5-time-selection-clocks</code> component, which
+ * is used in <code>ui5-time-picker</code> component respectively.
+ *
+ * <h3>Usage</h3>
+ *
+ * <code>ui5-time-picker-clock</code> can display hours, minutes or seconds
+ *
+ * <h3>ES6 Module Import</h3>
+ *
+ * <code>import "@ui5/webcomponents/dist/TimePickerClock.js";</code>
+ *
+ * @constructor
+ * @author SAP SE
+ * @alias sap.ui.webc.main.TimePickerClock
+ * @extends sap.ui.webc.base.UI5Element
+ * @tagname ui5-time-picker-clock
+ * @since 1.14.0
+ * @private
+ */
+@customElement({
+	tag: "ui5-time-picker-clock",
+	renderer: litRender,
+	styles: TimePickerClockCss,
+	template: TimePickerClockTemplate,
+})
+
+/**
+ * Fired when a value of clock is changed.
+ *
+ * @event sap.ui.webc.main.TimePickerClock#change
+ * @param { integer } value The new <code>value</code> of the clock.
+ * @param { string } stringValue The new <code>value</code> of the clock, as string, zero-prepended when necessary.
+ * @param { boolean } finalChange <code>true</code> when a value is selected and confirmed, <code>false</code> when a value is only selected but not confirmed.
+ */
+@event("change", {
+	detail: {
+		value: { type: Integer },
+		stringValue: { type: String },
+		finalChange: { type: Boolean },
+	},
+})
+
+class TimePickerClock extends UI5Element {
+	/**
+	 * Determines whether the component is displayed as disabled.
+	 *
+	 * @name sap.ui.webc.main.TimePickerClock.prototype.disabled
+	 * @type {boolean}
+	 * @defaultvalue false
+	 */
+	@property({ type: Boolean })
+	disabled!: boolean;
+
+	/**
+	 * Determines whether the component is active (visible).
+	 *
+	 * @name sap.ui.webc.main.TimePickerClock.prototype.active
+	 * @type {boolean}
+	 * @defaultvalue false
+	 */
+	@property({ type: Boolean })
+	active!: boolean;
+
+	/**
+	 * Minimum item value for the outer circle of the clock.
+	 *
+	 * @name sap.ui.webc.main.TimePickerClock.prototype.itemMin
+	 * @type {integer}
+	 * @defaultvalue -1
+	 */
+	@property({ validator: Integer, defaultValue: -1 })
+	itemMin!: number;
+
+	/**
+	 * Maximum item value for the outer circle of the clock.
+	 *
+	 * @name sap.ui.webc.main.TimePickerClock.prototype.itemMax
+	 * @type {integer}
+	 * @defaultvalue -1
+	 */
+	@property({ validator: Integer, defaultValue: -1 })
+	itemMax!: number;
+
+	/**
+	 * If set to <code>true</code>, an inner circle is displayed.
+	 * The first item value of the inner circle will be itemMax + 1
+	 *
+	 * @name sap.ui.webc.main.TimePickerClock.prototype.innerItems
+	 * @type {boolean}
+	 * @defaultvalue false
+	 */
+	@property({ type: Boolean })
+	innerItems!: boolean;
```

Comment:
`innerItems` sounds like a slot of items, for a boolean `"show/hideInnerCircle"` is more appropriate

Reply:
renamed to showInnerCircle